### DISCUSSION TITLE
Switch from mpp to mpp_new on ollie

### DISCUSSION THREAD

**Discussion by fernandadialzira** [esm-tools/esm_tools#621] [HPCs] [answered]
We have received some e-mails about the new partitions for running MPI jobs on ollie, the so-called mpp_new. In the [Confluences Page](https://spaces.awi.de/display/HELP/Partitions+mpp_new%2C+smp_new) it says that we should set this change by hand, and it is actually advised to be done. For this I have 2 main questions:

How do we do it by hand?
Do we need to cancel our current jobs and rerun them under the new partition, or it can be changed while our simulation is running?

**Comment by denizural** [comment]
I think this is a good question for @pgierz

**Comment by pgierz** [comment, marked as answer]
@denizural, indeed, the idea was to get @fernandadialzira to start to use the discussions page to promote the usage further. 

For the questions @fernandadialzira raised:

1. You can include the new partitions as follows:

```yaml
computer:
    partitions:
        compute:
            name:  "mpp_new"
```

Examples for FESOM standalone may be found here:
```
/work/ollie/pgierz/SciComp/HPC_Tests/
```

2. No, you can keep everything the same as it currently is and nothing will change (for now). There will be a major update of *all* nodes soon, but there will be a HPC email about that.

**Reply by pgierz** [reply]
And, to the last question: can we change it "on the fly": no, you will need to kill off your job, add the relevant lines to your config, and resubmit.
